examples/democoin/cmd/democli/main.go

```diff
@@ -38,6 +40,13 @@ func main() {
 	// get the codec
 	cdc := app.MakeCodec()
 
+	// Setup certain SDK config
+	config := sdk.GetConfig()
+	config.WithBech32PrefixForAccount("demoacc", "demopub")
```

Comment:
These should have `Set*` instead of `With*` as the prefix imo. We're mutating the configuration instead of returning a copy with a value set.

Reply:
Agree!